networking/ciao-cnci-agent/scripts/generate_cnci_cloud_image.sh

```diff
@@ -63,19 +63,18 @@ fi
 
 echo -e "\nMounting image: $image"
 tmpdir=$(mktemp -d)
-sudo modprobe nbd max_part=63
-sudo qemu-nbd --format=raw -c /dev/nbd0 "$image" 
 
 #it can take some time for the device to get created
 retry=0
+loop=''
 until [ $retry -ge 3 ]
```

Comment:
@obedmr do we need the retry. I used to have it before for qemu-nbd. I do not think the loop device needs the re-try?

besides that this is a nice change to lower dependencies.